### PR TITLE
feat(api): add /health endpoint to check workers

### PR DESCRIPTION
**Pull request by matthiasschaub**

### CODE REVIEW

**Review by jlink — APPROVED** [review]
As far as I understand celery and flask, it looks alright to me. Since this is an additional endpoint, it shouldn't break anything else, even if there is a bug in the code.

File: sketch_map_tool/routes.py [hunk 1]
```diff
@@ -220,6 +220,15 @@ def download(uuid: str, type_: REQUEST_TYPES) -> Response:
     return send_file(file, mimetype, download_name=download_name)
 
 
+@app.route("/api/ping")
+def ping():
+    """Ping Celery workers."""
+    result: list = celery_app.control.ping(timeout=1)
+    if result:
+        return {"status": "ok"}
+    return {"status": "fail"}
```
Comment (joker234) [inline sketch_map_tool/routes.py:229]: I think this still returns a `2xx` status code. Is this on purpose?

File: sketch_map_tool/routes.py [hunk 2]
```diff
@@ -220,6 +220,15 @@ def download(uuid: str, type_: REQUEST_TYPES) -> Response:
     return send_file(file, mimetype, download_name=download_name)
 
 
+@app.route("/api/ping")
```
Comment (joker234) [inline sketch_map_tool/routes.py:223]: Is `ping` the term you want to use? I often saw `health` or maybe `status` as endpoint name. The term `ping` also refers to ICMP requests and not necessarily to TCP/QUIC requests.

Comment (matthiasschaub) [reply sketch_map_tool/routes.py:223]: `/status` is already in use for querying the status of a certain task (e.g. map generation). `/health` I like but seems to encompass more than one ping to on part of the system. On the other hand, one can add new parts to the `health` endpoint over time.

Comment (matthiasschaub) [reply sketch_map_tool/routes.py:223]: renamed to `/health`

File: sketch_map_tool/routes.py (hunk 1 above)
Comment (matthiasschaub) [reply sketch_map_tool/routes.py:229]: Its on purpose. I do not know of any good alternative. The endpoint did what it is supposed to do, check the celery worker and reports on it. 
I think it is fine like this.

**Pull request merged**